airflow/providers/jenkins/operators/jenkins_job_trigger.py

```diff
@@ -153,9 +153,18 @@ def poll_job_in_queue(self, location: str, jenkins_server: Jenkins) -> int:
         # once it will be available in python-jenkins (v > 0.4.15)
         self.log.info('Polling jenkins queue at the url %s', location)
         while try_count < self.max_try_before_job_appears:
-            location_answer = jenkins_request_with_headers(
-                jenkins_server, Request(method='POST', url=location)
-            )
+            try:
+                location_answer = jenkins_request_with_headers(
+                    jenkins_server, Request(method='POST', url=location)
+                )
+            # we don't want to fail the operator, this will continue to poll
+            # until max_try_before_job_appears reached
+            except (HTTPError, JenkinsException) as ex:
+                self.log.info(f'polling failed, retry polling. Failure reason: {ex}')
```

Comment:
Do we want to use `warning` here instead? (No preference personally.)

Reply:
Either way though, this should use `self.log.xxx("Polling failed, retrying.", exc_info=True)` instead. Don’t pass the exception in manually.

Reply:
corrected, thanks for great comment!